specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -158,6 +158,145 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingActivityList": {
+      "get": {
+        "tags": [
+          "BillingActivity"
+        ],
+        "x-ms-examples": {
+          "BillingActivityList": {
+            "$ref": "./examples/BillingActivityList.json"
+          }
+        },
+        "operationId": "BillingActivityList_Get",
+        "description": "Gets billing activity logs for a billing account",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/startDateParameter"
+          },
+          {
+            "$ref": "#/parameters/endDateParameter"
+          },
+          {
+            "$ref": "#/parameters/activityLogEventCategoryParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BillingActivityList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingActivityLogs/{activityLogName}": {
+      "get": {
+        "tags": [
+          "BillingActivity"
+        ],
+        "x-ms-examples": {
+          "BillingActivityRelatedProperties": {
+            "$ref": "./examples/BillingActivityRelatedProperty.json"
+          }
+        },
+        "operationId": "BillingActivityLogsRelatedProperties_Get",
+        "description": "Gets detailed billing activity logs by activity log id and billing account name",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/billingAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/activityLogNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/BillingActivityRelatedProperty"
```

Comment:
the response to this API shoudl look like a resource (with a top level id, name, and type).  This is a GET on a single entity